### PR TITLE
EASY-1479: DDM to EMD should parse ISO 3166, and other bugs

### PR DESCRIPTION
**Pull request by rvanheest**
Fix a couple of bugs in the DDM -> EMD transformation:

* `ISO3166/dcterms:spatial` was not parsed at all
* accessRights handler needs to add a `schemeId=common.dcterms.accessrights` to the `BasicString`
* the polygon must also be parsed without `internal` or `description` fields

### CODE REVIEW

**Comment by rvanheest** [comment]
@DANS-KNAW/easy for review

File: src/main/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdHandlerMap.java [hunk 1]
```diff
@@ -56,6 +56,7 @@
 import nl.knaw.dans.pf.language.ddm.handlers.SubjectHandler;
 import nl.knaw.dans.pf.language.ddm.handlers.TermsRightsHolderHandler;
 import nl.knaw.dans.pf.language.ddm.handlers.TermsSpatialHandler;
+import nl.knaw.dans.pf.language.ddm.handlers.TermsSpatialIso3166Handler;
```
Comment (jo-pol) [inline src/main/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdHandlerMap.java]: Why did you add these imports?
